agent/consul/state/memdb_wrapper.go

```diff
@@ -0,0 +1,95 @@
+package state
+
+import (
+	"github.com/hashicorp/go-memdb"
+)
+
+// memDBWrapper is a thin shim over memdb.DB which forces all new tranactions to
```

Comment:
```suggestion
// memDBWrapper is a thin shim over memdb.DB which forces all new transactions to
```

Reply:
Turns out cSpell thinks "tranactions" is a valid word 🤷‍♂